<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>382</width>
       <height>600</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(0, 0, 0);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>382</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 100px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>C:/Users/admin/Pictures/Saved Pictures/2021_01_07_13_42_IMG_1257.JPG</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color:#EFEBF2;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">background-color:#EFEBF2;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:#EFEBF2;</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>28</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>Player dou</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="styleSheet">
            <string notr="true">background-color:#EFEBF2;</string>
           </property>
           <widget class="QLineEdit" name="edt_username">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>80</y>
              <width>341</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
background-color: white;
border: 1px solid rgb(180, 180, 180);
border-radius: 10px;
}
</string>
            </property>
            <property name="placeholderText">
             <string>Username</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="edt_password">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>160</y>
              <width>341</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
background-color: white;
border: 1px solid rgb(180, 180, 180);
border-radius: 10px;
}
</string>
            </property>
            <property name="placeholderText">
             <string>Password</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_login">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>250</y>
              <width>221</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color:#F95454;
border-radius: 10px;

</string>
            </property>
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
           <widget class="QLabel" name="lb_createAccount">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>310</y>
              <width>91</width>
              <height>16</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color:blue;
font-size: 14px;</string>
            </property>
            <property name="text">
             <string>Tạo tài khoản</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:#EFEBF2;</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
